README.md

```diff
@@ -203,13 +203,24 @@ passed a ``Featurizer`` object. ``deepchem`` provides a number of
 different subclasses of ``Featurizer`` for convenience:
 
 ### Performances
```

Comment:
Thanks for getting the model training times! Could you add in the loading-time for the graph-conv as well? (Since the fingerprint is different, loading time will likely be different).